libmscore/style.cpp

```diff
@@ -1992,31 +1992,39 @@ bool MStyle::readProperties(XmlReader& e)
                         set(idx, QVariant(e.readElementText()));
                   else if (!strcmp("Ms::Align", type)) {
                         QStringList sl = e.readElementText().split(',');
-                        if (sl.size() != 2) {
-                              qDebug("bad align text <%s>", qPrintable(e.readElementText()));
-                              return true;
+                        Align align;
+                        if (sl.size() == 1) {
+                              // This means align has been stored as a number instead of a
```

Comment:
Isn't it a bug? Did last changes in file format affect this? I mean it is interesting that the file format has been changed but the code has not been adapted here...

Reply:
I'm not sure, I'd guess that the new way of storing align is as a comma delimited list, of horizontal align and vertical align. When you click the 'Set as style' button, for some reason it gets set in a way that writes it as a number, which can still be read fine. So really, the real problem might be in the 'Set as style' button code, but my code does still fix the problem.

Reply:
@anatoly-os If you are speaking about those fileformat changes that were made during my GSoC project they changed nothing in styles at all. At least I didn't see a reason for such changes since I didn't try to compare styles.

Reply:
@jthistle Am I understand correctly, that this fix is required because of the changes in how styles are stored?

Reply:
Yes - alignment used to be stored as a number which could be converted to binary to set horizontal and vertical alignment, but now the '`halign,valign`' format is used. When you click the 'set as style' button next to alignment in the inspector, it writes it as a number for some reason. Saving the file, reloading it and saving it again leads to it being changed to the new format. So this fix doesn't really solve the root of the problem, but it does solve the problem.